lib/blockchain/common.js

```diff
@@ -99,7 +105,11 @@ function fetchAndInstall (coinRec) {
   es(`wget -q ${url}`)
   es(`tar -xzf ${downloadFile}`)
 
-  if (_.isEmpty(binaries.files)) {
+  if (binaries.build) {
+    es(`${__dirname}/${binaries.build.script} \`realpath ${binaries.build.folder}\``)
```

Comment:
This is confusing as normally you would expect build script to be relative to `build.folder`, not `__dirname`. I understand why it is how it is but it is still unexpected.

Reply:
I know, but have you any solution? That installation process is runned in `/tmp` folder where is downloaded Cardano SL source code, but i wanna to run our build script which is in Lamassu server folder...